lab-davidw/models/lure.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
One of your resources also must have: 'add an array of type mongoose.Schema.ObjectId with a ref to your new-model on your model from lab-13'. Per lab instructions